site2/website-next/scripts/code.json

```diff
@@ -0,0 +1,501 @@
+    "message": "Apache Pulsar downloads"
+  },
+  "Release notes": {
+    "message": "Release notes"
+  },
+  "Current version (Stable)": {
+    "message": "Current version (Stable)"
+  },
+  "Tiered storage offloaders": {
+    "message": "Tiered storage offloaders"
+  },
+  "Pulsar IO connectors": {
+    "message": "Pulsar IO connectors"
+  },
+  "Release Integrity": {
+    "message": "Release Integrity"
+  },
+  "Getting started": {
+    "message": "Getting started"
+  },
+  "Once you've downloaded a Pulsar release, instructions on getting up and running with a standalone cluster that you can run on your laptop can be found in the": {
+    "message": "Once you've downloaded a Pulsar release, instructions on getting up and running with a standalone cluster that you can run on your laptop can be found in the"
+  },
+  "Run Pulsar locally": {
+    "message": "Run Pulsar locally"
+  },
+  "tutorial": {
+    "message": "tutorial"
+  },
+  "If you need to connect to an existing Pulsar cluster or instance using an officially supported client, see the client docs for these languages:": {
+    "message": "If you need to connect to an existing Pulsar cluster or instance using an officially supported client, see the client docs for these languages:"
+  },
+  "Older releases": {
+    "message": "Older releases"
+  },
+  "Pulsar Adapters": {
+    "message": "Pulsar Adapters"
+  },
+  "Pulsar Adapters are available on Maven Central, there is no binary package.": {
+    "message": "Pulsar Adapters are available on Maven Central, there is no binary package."
+  },
+  "Apache Pulsar Manager downloads": {
+    "message": "Apache Pulsar Manager downloads"
+  },
+  "Pulsar Manager older releases": {
+    "message": "Pulsar Manager older releases"
+  },
+  "Latest Stable Version": {
+    "message": "Latest Stable Version"
+  },
+  "Latest Version": {
+    "message": "Latest Version"
+  },
+  "Here you can find the latest documentation and unreleased code.": {
+    "message": "Here you can find the latest documentation and unreleased code."
+  },
+  "Past Version": {
+    "message": "Past Version"
+  },
+  "Here you can find documentation for previous versions of Apache Pulsar.": {
+    "message": "Here you can find documentation for previous versions of Apache Pulsar."
+  },
+  "theme.docs.versions.unreleasedVersionLabel": {
+    "message": "This documentation is for an unreleased {siteTitle} {versionLabel} version of Apache Pulsar. ",
+    "description": "The label used to tell the user that he's browsing an unreleased doc version"
+  },
+  "theme.docs.versions.unmaintainedVersionLabel": {
+    "message": "This documentation is for an out-of-date {siteTitle} {versionLabel} version of Apache Pulsar. ",
+    "description": "The label used to tell the user that he's browsing an unmaintained doc version"
+  },
+  "theme.docs.versions.latestVersionSuggestionLabel": {
+    "message": "We recommend you use the {latestVersionLink} ({versionLabel}).",
```

Comment:
In technical writing:

- Avoid first-person pronouns. Details see https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.gfwms5zehcak. 
- Avoid using "recommend" if possible. A phrase such as “We recommend that you take the following action” could create a potential marketing or legal problem